src/App.tsx

```diff
@@ -26,6 +26,9 @@ function App() {
             ❤️ <Provincle /> ❓
           </a>
         </span>
+        <br /> {/* bottom spaceholder for Android browser */}
```

Comment:
Not sure about this. I wouldn't worry about some browsers not showing the whole height of the page. This looks odd to me on DDG, and Worldle didn't do anything about weird Chrome, Brave, etc. displays.

Reply:
right. Actually it would require a 3xBR.
And I found it's not the browser, but the OS (Android) doing that.
Even though Worldle ignores this I do find annoying that the footer is not visible. Let's move this issue later, I revert this

Reply:
But DDG displays it properly for me on Android. Anyway, let's talk about this later on, see if we want to do anything about it.

Reply:
removed. 
You are right DDG good, chrome not good :(